src/components/Text.tsx

```diff
@@ -0,0 +1,179 @@
+import {
+  TGradientColorStops,
+  TTextColor,
+  backgroundClip,
+  backgroundImage,
+  classnames,
+  dropShadow,
+  fontFamily,
+  fontSize,
+  fontWeight,
+  gradientColorStops,
+  letterSpacing,
+  lineHeight,
+  textAlign,
+  textColor,
+  textOverflow,
+  textTransform,
+} from 'classnames/tailwind'
+import ChildrenProp from 'models/ChildrenProp'
+import classNamesToString from 'helpers/classNamesToString'
+
+const foldText = (
+  color?: TTextColor,
+  gradientFrom?: TGradientColorStops,
+  gradientTo?: TGradientColorStops
+) =>
+  classnames(
+    fontFamily('font-primary'),
+    fontSize('text-2xl', 'md:text-3xl'),
+    lineHeight('leading-8', 'md:leading-10'),
+    fontWeight('font-bold'),
+    textColor(
+      gradientFrom && gradientTo ? 'text-transparent' : color || 'text-accent'
+    ),
+    backgroundImage(
+      gradientFrom && gradientTo ? 'bg-gradient-to-r' : undefined
+    ),
+    backgroundClip(gradientFrom && gradientTo ? 'bg-clip-text' : undefined),
+    gradientColorStops(gradientFrom, gradientTo)
+  )
+export function FoldText({
+  color,
+  gradientFrom,
+  gradientTo,
+  children,
+}: ChildrenProp & {
+  color?: TTextColor
+  gradientFrom?: TGradientColorStops
+  gradientTo?: TGradientColorStops
+}) {
+  return (
+    <span className={foldText(color, gradientFrom, gradientTo)}>
+      {children}
+    </span>
+  )
+}
+
+const largeStrokeTextAccent = classnames(
+  textColor('text-transparent'),
+  textOverflow('text-clip'),
+  fontWeight('font-bold'),
+  textAlign('text-justify'),
+  textTransform('uppercase'),
+  fontFamily('font-primary'),
+  fontSize('text-5xl', 'md:text-8xl'),
+  lineHeight('leading-10', 'md:leading-12')
+)
+export function LargeStrokeText({
+  small,
+  children,
+}: ChildrenProp & { small?: boolean }) {
+  return (
+    <h1
+      className={classNamesToString([
+        largeStrokeTextAccent,
+        small ? 'strokeTextSmall' : 'strokeText',
+      ])}
+    >
+      {children}
+    </h1>
+  )
+}
+
+const bodyText = (small?: boolean, bold?: boolean) =>
+  classnames(
+    textColor('text-formal-accent'),
+    lineHeight(small ? 'leading-5' : 'leading-6'),
+    fontSize(small ? 'text-sm' : 'text-base'),
+    fontWeight(bold ? 'font-bold' : 'font-normal')
+  )
+export function BodyText({
+  bold,
+  small,
+  children,
+}: ChildrenProp & { bold?: boolean; small?: boolean }) {
+  return <p className={bodyText(small, bold)}>{children}</p>
+}
+
+const accentText = (color: TTextColor, primary?: boolean) =>
+  classnames(textColor(color), primaryAccentText(primary))
+const primaryAccentText = (primary?: boolean) =>
+  primary
+    ? classnames(
+        fontFamily('font-primary'),
+        fontWeight('font-bold'),
+        lineHeight('leading-11'),
+        fontSize('text-4xl')
```

Comment:
I think that accent text may inherit some properties from `BodyText`, for example, you have such a string: 

This is _plain_ string

Which will look like: 
```TS
<BodyText> This is <AccentText>Plain</AccentText> string </BodyText>
```